Wiki.md

```diff
@@ -1,85 +1,92 @@
-## Intro
+# Error Prone Checker
+
 [Error Prone](https://github.com/google/error-prone) check to detect missing AutoDispose scope within defined scoped elements.
 
 ## Installation
 
 Here are sample configurations which pulls in both the ErrorProne and the AutoDispose check.
+
 ### Gradle
+
 ```gradle
-  buildscript {
-    repositories {
-      maven {
-        url "https://plugins.gradle.org/m2/"
-      }
+buildscript {
+  repositories {
+    maven {
+      url "https://plugins.gradle.org/m2/"
     }
   }
-  
-  plugins {
-    // we assume you are already using the Java plugin
-    id "net.ltgt.apt" version "0.13"
-    id "net.ltgt.errorprone" version "0.0.13"
-  }
-  
-  dependencies {
-    apt "com.uber.autodispose:autodispose-error-prone-checker:x.y.z" // where x.y.z is the latest version.
-  
-    errorprone "com.google.errorprone:error_prone_core:2.1.3"
-  }
-  
-  tasks.withType(JavaCompile) {
-    // Only if you want to support custom configuration
-    // Below is a sample configuration which include Conductor and Activity
-    options.compilerArgs += ["-XepOpt:AutoDisposeLeakCheck"
-                                         + "=com.bluelinelabs.conductor.Controller,android.app.Activity"]
-  }
+}
+
+plugins {
+  // we assume you are already using the Java plugin
+  id "net.ltgt.apt" version "0.13"
+  id "net.ltgt.errorprone" version "0.0.13"
+}
+
+dependencies {
+  apt "com.uber.autodispose:autodispose-error-prone-checker:x.y.z" // where x.y.z is the latest version.
+
+  errorprone "com.google.errorprone:error_prone_core:2.1.3"
+}
+
+tasks.withType(JavaCompile) {
```

Comment:
Does this need to be qualified?

Reply:
It works without that also but the flag will be available for all the checks but this is better - https://github.com/uber/AutoDispose/pull/162/commits/3d7d9056f1fe21a82396608db903c2c29fd6007c.

Reply:
I meant `JavaCompile`

Reply:
It doesn't need to be. Nullaway is configured like this - https://github.com/uber/NullAway

Reply:
:+1: